web_ui/frontend/components/Namespace/DeniedCard.tsx

```diff
@@ -0,0 +1,160 @@
+                                <>
+                                    <Tooltip title={"Delete Registration"}>
+                                        <IconButton
+                                            sx={{bgcolor: "#ff00001a", mx: 1}}
+                                            color={"error"}
+                                            onClick={async (e) => {
+                                                e.stopPropagation()
+                                                try {
+                                                    await deleteNamespace(namespace.id)
+                                                    await mutate("getNamespaces")
+                                                } catch (e) {
+                                                    if(e instanceof Error){
+                                                        setAlert({severity: "error", message: e.message})
+                                                    }
+                                                }
+                                            }}
+                                        >
+                                            <Delete/>
+                                        </IconButton>
+                                    </Tooltip>
+                                    <Tooltip title={"Approve Registration"}>
+                                        <IconButton
+                                            sx={{bgcolor: "#2e7d3224", mx: 1}}
+                                            color={"success"}
+                                            onClick={async (e) => {
+                                                e.stopPropagation()
+                                                try {
+                                                    await approveNamespace(namespace.id)
+                                                    await mutate("getNamespaces")
+                                                } catch (e) {
+                                                    if(e instanceof Error){
+                                                        setAlert({severity: "error", message: e.message})
+                                                    }
+                                                }
+                                            }}
+                                        >
+                                            <Check/>
+                                        </IconButton>
+                                    </Tooltip>
+                                </>
+                            }
+                        </Box>
+                    </Box>
+                </Box>
+                <Box ref={ref}>
+                    <InformationDropdown adminMetadata={namespace.admin_metadata} transition={transition} parentRef={ref}/>
+                </Box>
+            </Box>
+            <AlertPortal alert={alert} onClose={() => setAlert(undefined)} />
```

Comment:
I didn't see any alert when I successfully deleted a namespace, but we do if we approve a namespace in the dashboard view. Can we give user a successful message alert here?

Reply:
@haoming29 Going to start transitioning to making success states less intrusive. On success the card now turns green before it disappears rather then also popping up a alert.